src/components/common/animations/Base.js

```diff
@@ -1,10 +1,17 @@
 import React from 'react'
 import { cloneDeep, once } from 'lodash'
-import { isMobileNative } from '../../../lib/utils/platform'
+import { AppState } from 'react-native'
+import { isIOSNative, isMobileNative } from '../../../lib/utils/platform'
 
 class AnimationBase extends React.Component {
   componentDidMount() {
     this.initAnimation()
+
+    if (!isIOSNative) {
+      return
+    }
+
+    this.subscription = AppState.addEventListener('change', this.handleAppStateChange)
```

Comment:
that doesnt seem right, what is this code purpose on android? how would that not change the expected behaviour on iOS?

Reply:
It doesn't affect android, it fixes reset of animation progress on app state change on ios

Reply:
can you explain, why does animation is being reset on iOS?

Reply:
it is lottie-ios issue, and it hasn't been fixed yet. There is an issue on github, which suggests this fix. Also using Animated.Value and progress prop can fix this issue totally, so will implement it today.